spec/superset/guest_token_spec.rb

```diff
@@ -56,5 +56,24 @@
         )
       end
     end
+
+    context 'with rls clause' do
+      before { allow(subject).to receive(:rls_clause).and_return(rls_clause) }
+      let(:user) { nil }
+      let(:rls_clause) { [{ "clause": "publisher = 'Nintendo'" }] }
```

Comment:
Is it valid to include a scenario where `rls_clause` is nil, empty or somehow invalid (empty hash)?

Reply:
Sure will add test cases for those scenario as well.